documentation/tutorials/kaggle_beginner_example_classification.ipynb

```diff
@@ -0,0 +1,2105 @@
+{
```

Comment:
<p>"Load data" or "Load the dataset" instead of "Data Loading".</p><p> </p><p>See the <a href="https://developers.google.com/style/headings#heading-and-title-text" target="_blank">style guide</a> for the specifics on how to word a title.</p><p> </p><p>Also "using the TensorFlow Dataset to read the files may be better suited" should be "using TensorFlow Datasets to...", or if there is "<strong>a</strong> tensorflow dataset", then link to it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/decision-forests/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>